adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralBannerAdLoader.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+@interface GADMAdapterMintegralBannerAdLoader : NSObject<GADMediationBannerAd>
+
+/// Asks the receiver to render the ad configuration.
+- (void)loadBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguration *)adConfiguration
```

Comment:
Can you rename this to `loadBannerAdForAdConfiguration`

Reply:
OK, I've made changes and committed.

Reply:
OK, I've made changes and committed.